exercises/pov/src/POV.hs

```diff
@@ -3,7 +3,7 @@ module POV (fromPOV, tracePathBetween) where
 import Data.Tree (Tree)
 
 fromPOV :: Eq a => a -> Tree a -> Maybe (Tree a)
-fromPOV = error "You need to implement this function."
+fromPOV node tree = error "You need to implement this function."
 
 tracePathBetween :: Eq a => a -> a -> Tree a -> Maybe [a]
-tracePathBetween = error "You need to implement this function."
+tracePathBetween nodeA nodeB tree = error "You need to implement this function."
```

Comment:
Happy to make the changes. As the Haskell's POV page is broken, I checked the instructions for [Ruby](http://exercism.io/exercises/ruby/pov/readme), which says:

>This lets us more simply describe the paths between two nodes. So for example the path from 6-9 (which in the first graph goes up to the root and then down to a different leaf node) can be seen to follow the path 6-2-0-3-9
This exercise involves taking an input graph and re-orientating it from the point of view of one of the nodes.
 
Also, isn't a Leaf also considered a _node_ (often called _leaf node_) ?